cmd/cli/plugin/managementcluster/create.go

```diff
@@ -135,6 +141,9 @@ func init() {
 	createCmd.Flags().BoolVar(&iro.forceConfigUpdate, "force-config-update", false, "Force an update of all configuration files in ${HOME}/.config/tanzu/tkg/bom and ${HOME}/.tanzu/tkg/compatibility")
 
 	createCmd.Flags().SetNormalizeFunc(aliasNormalizeFunc)
+
+	createCmd.Flags().StringVarP(&iro.additionalManifests, "additional-manifests", "", "", "Additional manifests to be applied to the bootstrap cluster")
+	createCmd.Flags().StringVarP(&iro.additionalManifestsNamespace, "additional-manifests-namespace", "", "", "Namespace of the additional manifest")
```

Comment:
```suggestion
	createCmd.Flags().StringVarP(&iro.additionalManifestsNamespace, "additional-manifests-namespace", "", "", "Namespace of the additional manifest")
```

For the purposes of management-cluster plugin, would it ever anything but `tkg-system` since the management cluster MUST be created in that namespace.

Reply:
yep, statement is true. i am thinking renaming the flag to `--additional-tkg-system-manifests` and drop the other flag for specifying the NS